sdk/core/Azure.Core/src/DynamicData/MutableJsonDocument.cs

```diff
@@ -47,16 +47,34 @@ public MutableJsonElement RootElement
         /// <param name="format">A format string indicating the format to use when writing the document.</param>
         /// <exception cref="ArgumentNullException">The <paramref name="stream"/> parameter is <see langword="null"/>.</exception>
         /// <exception cref="FormatException">Thrown if an unsupported value is passed for format.</exception>
-        /// <remarks>The value of <paramref name="format"/> can be default or 'J' to write the document as JSON.</remarks>
-        public void WriteTo(Stream stream, StandardFormat format = default)
+        /// <remarks>The value of <paramref name="format"/> can be default or 'J' to write the document as JSON, or 'P' to write the changes as JSON Merge Patch.</remarks>
+        public void WriteTo(Stream stream, string? format = default)
         {
             Argument.AssertNotNull(stream, nameof(stream));
+            ValidateFormat(format);
 
-            if (format != default && format.Symbol != 'J')
+            switch (format)
             {
-                throw new FormatException($"Unsupported format {format.Symbol}. Supported formats are: 'J' - JSON.");
+                case "P":
+                    WritePatch(stream);
+                    break;
+                case "J":
+                default:
```

Comment:
Same here as a previous comment: should we really fall back to "J" internally? Seems like we'd want to know where code is misbehaving - especially internally.